src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/kafka/statusendring/KafkaStatusendring.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.syfo.oppfolgingstilfelle.bit.kafka.statusendring
+
+import java.time.OffsetDateTime
+
+const val STATUS_APEN = "APEN"
```

Comment:
Kunne man definert en enum for disse og brukt den som type på statusEvent?

Reply:
Ja, bra inspill - skal se på det 👍